src/ansys/scade/apitools/auto_scade_env.py

```diff
@@ -58,7 +59,7 @@ def _get_scade_dirs(min='00.0', max='99.9'):
     if platform.system() == 'Windows':
         for company in 'Esterel Technologies', 'Ansys Inc':
             try:
-                hklm = reg.OpenKey(reg.HKEY_LOCAL_MACHINE, 'SOFTWARE\%s\SCADE' % company)
+                hklm = reg.OpenKey(reg.HKEY_LOCAL_MACHINE, r'SOFTWARE\%s\SCADE' % company)
```

Comment:
```suggestion
                hklm = reg.OpenKey(reg.HKEY_LOCAL_MACHINE, r'SOFTWARE\{}\SCADE'.format(company))
```
We may use this instead. There is nothing wrong with the original approach, it's just that this is considered best practice within the python community.

Reply:
I understand `printf` style is out-dated, I'm wondering whether we should `format` of f-strings, as advised in other reviews.

Reply:
Both are acceptable. In this specific case, I think using `format` is more readable as it separates the raw-string declaration from the formatting. Using f-string in this this specific case would have to be something like `hklm = reg.OpenKey(reg.HKEY_LOCAL_MACHINE, rf'SOFTWARE\{company}\SCADE')`. I don't know how readable `rf` is.

Reply:
Thanks.